scripts/vep_rule.py

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
This looks like it's duplicated in the snakefile.

Either include this file with `include` or remove it.  Duplication is bad.